apps/chat/src/components/Layout.tsx

```diff
@@ -19,13 +21,23 @@ import {
 } from '@/src/store/settings/settings.reducers';
 import { UIActions } from '@/src/store/ui/ui.reducers';
 
+const getPageType = (route?: string) => {
+  switch (route) {
+    case '/marketplace':
```

Comment:
what do you thing about moving it to some config/routes.ts?

Reply:
let's do this in further improvements after discussion, I don't know the right way to do it right now